src/components/Popups/PopupReply.tsx

```diff
@@ -60,6 +54,27 @@ export const options: Partial<Popper.Options> = {
     ],
 };
 
+const getModifiers = (): Partial<Popper.Options> => {
+    const { modifiers: defaultModifiers = [] } = options;
```

Comment:
It looks like `options` is hardcoded in this file. Can `modifiers` here be undefined?

Reply:
According to the type for `Partial` it seems like it could be `undefined`

Reply:
Gotcha. Should we update the type to be `Pick` rather than `Partial`, in that case?